services/tutorials/ethereum/send-a-transaction/use-web3.js.md

```diff
@@ -2,29 +2,43 @@
 description: Send a transaction using Web3.js.
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Use web3.js
 
-In this tutorial, you'll send a regular transaction from one account to another using the Web3 JavaScript library.
+In this tutorial, you'll send a regular transaction of 0.001 ETH from one account to another using the Web3 JavaScript library.
 
 ## Prerequisites
 
-- A [Web3 project](../../../../../developer-tools/dashboard/get-started/create-api/) on Infura
+- A [Web3 project](../../../api/getting-started.md) on Infura
 - [Node.js installed](https://nodejs.org/en/download/)
-- An Ethereum account for testing purposes
+- An Ethereum account
 
 :::info
 
-You can use [MetaMask](https://metamask.io) or similar to create an Ethereum account for testing purposes.
+Use [MetaMask](https://metamask.io) or similar to create an Ethereum account for testing.
 
 :::
 
 ## Steps
 
-### 1. Fund your Ethereum account
+### 1. Select your network and verify funds
+
+<Tabs>
+  <TabItem value="View" label="Sepolia" default>
 
-[Use the Infura faucet to load testnet ETH](https://www.infura.io/faucet) on your Ethereum account for the Sepolia network.
+Verify that your wallet has testnet ETH for the Sepolia network. You may use the [Infura faucet](https://www.infura.io/faucet) to top up. 
 
-If using a network other than Sepolia, ensure you [update your environment file](#4-create-the-env-file) with the network name.
+</TabItem>
+<TabItem value="Retrieve" label="Alternative network" default>
+
+To use an alternative network:
+- Update your [environment file](#4-create-the-env-file) with the alternative network name 
```

Comment:
I'm not sure, but should this be numbered as a procedure? I know it does not have to ordered, but maybe this is ordinal?